packages/grafana-ui/src/themes/ThemeContext.tsx

```diff
@@ -3,19 +3,29 @@ import hoistNonReactStatics from 'hoist-non-react-statics';
 
 import { getTheme } from './getTheme';
 import { Themeable } from '../types/theme';
-import { GrafanaThemeType } from '@grafana/data';
+import { GrafanaTheme, GrafanaThemeType } from '@grafana/data';
 
 type Omit<T, K> = Pick<T, Exclude<keyof T, K>>;
 type Subtract<T, K> = Omit<T, keyof K>;
 
+/**
+ * Mock used in tests tests
```

Comment:
```suggestion
 * Mock used in tests
```

I assume this is a `tests` too much?